addons/mail/models/mail_thread.py

```diff
@@ -288,14 +284,16 @@ def create(self, vals_list):
         if not self._context.get('mail_create_nolog'):
             threads_no_subtype = self.env[self._name]
             for thread in threads:
-                body = thread._get_creation_message()
                 subtype = thread._creation_subtype()
                 if subtype:  # if we have a subtype, post message to notify users from _message_auto_subscribe
-                    thread.sudo().message_post(body=body, subtype_id=subtype.id, author_id=self.env.user.partner_id.id)
+                    thread.sudo().message_post(subtype_id=subtype.id, author_id=self.env.user.partner_id.id)
                 else:
                     threads_no_subtype += thread
             if threads_no_subtype:
-                threads_no_subtype._message_log_batch(bodies={t.id: body for t in threads_no_subtype})
+                bodies = dict(
+                    (thread.id, thread._creation_message())
+                    for thread in threads_no_subtype)
+                threads_no_subtype._message_log_batch(bodies=bodies)
```

Comment:
I wish we could get rid of those body too but I guess that might be asking too much.